Core/include/Acts/Seeding/SeedfinderConfig.hpp

```diff
@@ -65,6 +65,9 @@ struct SeedfinderConfig {
   // parameters for forward seed confirmation
   SeedConfirmationRange forwardSeedConfirmationRange;
 
+  // cut to the maximum value of delta z between SPs
+  float deltaZMax = std::numeric_limits<float>::max() * Acts::UnitConstants::mm;
```

Comment:
float is relatively safe in that respect, as it would just go to infinity if mm were >1
setting it to std::numeric_limits<float>::infinity() also works with the multiplication and is unambiguous.